symfony/plugins/orangehrmMaintenancePlugin/test/AccessStrategy/FormatValue/FormatWithBase64EncodeTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA
+ */
+
+namespace OrangeHRM\Tests\Maintenance\AccessStrategy\FormatValue;
+
+use OrangeHRM\Config\Config;
+use OrangeHRM\Maintenance\AccessStrategy\FormatValue\FormatWithBase64Encode;
+use OrangeHRM\Tests\Util\KernelTestCase;
+use OrangeHRM\Tests\Util\TestDataService;
+
+class FormatWithBase64EncodeTest extends KernelTestCase
+{
+    private string $fixture;
+    private FormatWithBase64Encode $formatWithBase64Encode;
+
+    protected function setUp(): void
+    {
+        $this->fixture=Config::get(Config::PLUGINS_DIR).'/orangehrmMaintenancePlugin/test/fixtures/EmployeeDao.yml';
+        TestDataService::populate($this->fixture);
+        $this->formatWithBase64Encode = new FormatWithBase64Encode();
+    }
+
+    public function testGetFormattedValue()
+    {
+        $this->assertEquals("", $this->formatWithBase64Encode->getFormattedValue(""));
```

Comment:
maybe you can add another test case here with a string and expect the base 64 value
`$this->assertEquals("QWppaGFyYW4=", $this->formatWithBase64Encode->getFormattedValue("Ajiharan"));`